variables.tf

```
@@ -39,3 +39,9 @@ variable "create_iam_access_key" {
   description = "Whether or not to create IAM access keys"
   default     = true
 }
+
+variable "write_keys_to_ssm_enabled" {
```

Why not simply this ?
```suggestion
variable "ssm_enabled" {
```